optaplanner-examples/src/main/java/org/optaplanner/examples/batchscheduling/domain/solver/RoutePathStrengthWeightFactory.java

```diff
@@ -0,0 +1,26 @@
+package org.optaplanner.examples.batchscheduling.domain.solver;
+
+import java.io.Serializable;
+import java.util.Comparator;
+
+import org.optaplanner.examples.batchscheduling.domain.AllocationPath;
+
+public class RoutePathStrengthWeightFactory implements Comparator<AllocationPath>, Serializable {
+
+    private static final long serialVersionUID = 2970371491784586248L;
```

Comment:
Even though I understand that the `Serializable` is recommended there, OptaPlanner doesn't really support serialization. So feel free to remove the use of the interface, incl. the `UID`s.